tests/openvino/native/quantization/test_weights_compression.py

```diff
@@ -297,8 +297,20 @@ def test_mixed_precision(mode, all_layers, ratio, ref_ids, mocker):
 
 @pytest.mark.parametrize("metric", DATA_BASED_SENSITIVITY_METRICS)
 def test_gather_in_4_bit_if_all_layers_with_data(metric):
-    model = IntegerModel().ov_model
-    dataset = Dataset([np.arange(7).reshape(1, 7, 1)])
+    dim1 = 2  # sequence length dimension
+    dim2 = 7
+    max_input_value = 6
+    model = IntegerModel(dim1=dim1, dim2=dim2, max_input_value=max_input_value, add_batch_dimension=True).ov_model
```

Comment:
This test was refactored, because it was not strictly correct before as data based criteria were receiving 1D activations